SCSS/Theme Rewrite/custom-css/callouts/_callout-type-timeline.scss

```diff
@@ -110,4 +114,4 @@
     // &[data-callout-metadata~="b-8"] {   padding-bottom: var(--med-tall); }
     // &[data-callout-metadata~="b-9"] {   padding-bottom: var(--tall); }
     // &[data-callout-metadata~="b-10"] {  padding-bottom: 750px; }
-}
\ No newline at end of file
+}
```

Comment:
Not sure what happened here - this was done automatically by github's web editor